### DISCUSSION TITLE
Simulation of a terrestrial exoplanet climate

### DISCUSSION THREAD

**Discussion by dennissergeev** [pyvista/pyvista#5043] [Show and tell]
It was [suggested](https://github.com/bjlittle/geovista/discussions/309#discussioncomment-7241832) to me to post the cover figure I made using `geovista` for my latest [paper](https://doi.org/10.5194/gmd-16-5601-2023) (published today :tada:).

![](https://gmd.copernicus.org/articles/16/5601/2023/gmd-16-5601-2023-avatar-web.png)

Just wanted to say massive thanks again to the creators and maintainers of PyVista and GeoVista!

**Comment by tkoyama010** [comment]
Thanks @dennissergeev ! We are glad that our project contributed to your paper.
